src/FileUpload/FileNameGenerator/Simple.php

```diff
@@ -56,11 +56,15 @@ public function getFileName($source_name, $type, $tmp_name, $index, $content_ran
      */
     protected function getUniqueFilename($name, $type, $index, $content_range)
     {
+        if (! is_array($content_range)) {
+            $content_range = [0];
+        }
+
         while ($this->filesystem->isDir($this->pathresolver->getUploadPath($name))) {
             $name = $this->pathresolver->upcountName($name);
         }
 
-        $uploaded_bytes = Util::fixIntegerOverflow(intval($content_range[1]));
+        $uploaded_bytes = Util::fixIntegerOverflow(intval($content_range[1] ?? $content_range[0]));
```

Comment:
Using the `??` syntax to check whether `$content_range[1]` is undefined.

Getting the `$content_range[0]` variable directly if the `$content_range[1]` is undefined.